Doc/library/contextlib.rst

```diff
@@ -502,11 +502,13 @@ Functions and classes provided:
           # the with statement, even if attempts to open files later
           # in the list raise an exception
 
+   The :meth:`__enter__` method returns the :class:`ExitStack` instance, and 
+   performs no additional operations.
+
    Each instance maintains a stack of registered callbacks that are called in
    reverse order when the instance is closed (either explicitly or implicitly
    at the end of a :keyword:`with` statement). Note that callbacks are *not*
    invoked implicitly when the context stack instance is garbage collected.
-
```

Comment:
Please put this line back

Reply:
Added the line in the latest commit. please review.

Reply:
I cannot see any trailing spaces. Are you able to locate which line it is?